### DISCUSSION TITLE
Ripples wallpaper: weather forecast / clock panel is ill placed while using double monitor systsem and panel animation is too busy

### DISCUSSION THREAD

**Discussion by Ziggizag** [rocksdanister/lively#972] [General]
Hi!

"Ripples" is a very nice, quiet wallpaper I like very much with it's underwater look but I observe three problems with this:

1. I use double monitor system. When I activate weather forecast / clock panel it centers exactly on screen bezels. While the wallpaper is stretched across two monitors panel should center either on the first or second one (it should be selectable).
2. It should be possible to select transparent foreground for weather forecast / clock panel. Transparency level should be adjustable.
3. Foreground animation (ripples on panel) is too intensive and it simply does not match rest of the screen. It is obstructive and unnecessary.

- Optimally, weather / clock information should be integrated into background and should center on the selected screen. No foreground image is needed at all, in my humble opinion.

Because of these three issues I simply turned weather forecast / clock pane off completely though I would love to use it if implemented properly.

**Comment by rocksdanister** [comment]
Most of the included wallpapers are just html pages, you can try modifying it yourself and maybe share it with the community :)

The bundled wallpapers are just example showcase for the software and I try not to take wallpaper requests, instead just focus my time on the software side.

**Reply by Ziggizag** [reply]
You were right actually. It took me 15 minutes to edit your code to get satisfactory result. I have also found a better image (with appropriate size for my setup). Thank you very much
